cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -227,13 +227,15 @@ export class VmsPage extends CruResourcePo {
         image.setNameNsDescription(name, namespace);
         image.setBasics({url});
         image.save();
-        image.censorInColumn(name, 3, namespace, 4, 'Active', 2, { timeout: constants.timeout.uploadTimeout });
-        image.censorInColumn(name, 3, namespace, 4, 'Completed', 5);
-        image.censorInColumn(name, 3, namespace, 4, '16 MB', 6);
+        image.checkState(name, namespace);
       }
     })
   }
 
+  public checkState(name:string, namespace:string, state:string = 'Running') {
```

Comment:
`name:string, namespace:string, state:string = 'Running'`    =>  `name:string,  state:string = 'Running', namespace:string = false,`     

Maybe it would be better to switch places